utilities/build_envs.sh

```diff
@@ -18,7 +18,7 @@ if buildkite-agent meta-data exists BUILDKITE_JULIA_BRANCH; then
 fi
 
 # Determine JULIA_CPU_TARGETS for different architectures
-JUlIA_CPU_TARGETS=()
+JULIA_CPU_TARGETS=()
```

Comment:
How was this not failing??

Reply:
Because `bash` doesn't have problems with `+=` for an undefined variable.  I don't know if there's a mode to turn that into an error, `set -u` doesn't seem to help here.